[2025-01-13T21:09:49.644+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prepare_churn_dataset.create_table_task manual__2025-01-13T21:09:46.587070+00:00 [queued]>
[2025-01-13T21:09:49.650+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prepare_churn_dataset.create_table_task manual__2025-01-13T21:09:46.587070+00:00 [queued]>
[2025-01-13T21:09:49.651+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-01-13T21:09:49.661+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): create_table_task> on 2025-01-13 21:09:46.587070+00:00
[2025-01-13T21:09:49.668+0000] {standard_task_runner.py:57} INFO - Started process 73 to run task
[2025-01-13T21:09:49.671+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'prepare_churn_dataset', 'create_table_task', 'manual__2025-01-13T21:09:46.587070+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/churn.py', '--cfg-path', '/tmp/tmp0yf4_hel']
[2025-01-13T21:09:49.672+0000] {standard_task_runner.py:85} INFO - Job 146: Subtask create_table_task
[2025-01-13T21:09:49.714+0000] {task_command.py:416} INFO - Running <TaskInstance: prepare_churn_dataset.create_table_task manual__2025-01-13T21:09:46.587070+00:00 [running]> on host 9dba7ac9ecaf
[2025-01-13T21:09:49.775+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prepare_churn_dataset' AIRFLOW_CTX_TASK_ID='create_table_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T21:09:46.587070+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T21:09:46.587070+00:00'
[2025-01-13T21:09:49.777+0000] {base.py:73} INFO - Using connection ID 'destination_db' for task execution.
[2025-01-13T21:09:50.228+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-01-13T21:09:50.237+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_churn_dataset, task_id=create_table_task, execution_date=20250113T210946, start_date=20250113T210949, end_date=20250113T210950
[2025-01-13T21:09:50.286+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-13T21:09:50.303+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
